openlibrary/plugins/openlibrary/js/markdown-editor/index.js

```diff
@@ -0,0 +1,14 @@
+// unversioned.
+import '../../../../../vendor/js/wmd/jquery.wmd.js'
+
+/**
+ * Sets up Wikitext markdown editor interface inside $textarea
+ * @param {jQuery.Object} $textareas
+ */
+export function initMarkdownEditor($textareas) {
+    $textareas.wmd({
+        helpLink: '/help/markdown',
+        helpHoverTitle: 'Formatting Help',
+        helpTarget: '_new'
+    });
+}
```

Comment:
The selector `textarea.markdown` is also used in `ol.js`; can we move it here?

Reply:
Nice catch! Doing as we speak..

Reply:
https://github.com/internetarchive/openlibrary/pull/2371